### PR TITLE
Dashboard copy changes

### PR DESCRIPTION
**Pull request by shahthepro**

### CODE REVIEW

File: templates/dashboard.html [hunk 1]
```diff
@@ -98,7 +98,7 @@ <h3>{{gettext("Modeled vs. actual token release schedule")}}</h3>
         <div class="col-md-6">
           <h3>{{gettext("Locked Tokens")}}</h3>
           <div class="desc">
-            {{gettext("Fusce condimentum ultrices est quis dapibus. Phasellus massa tortor, ultricies a imperdiet in, dapibus ac ligula. Duis suscipit odio sed sagittis finibus. Curabitur justo neque, scelerisque eu velit interdum, finibus varius purus.")}}
+            {{gettext("Many of our investors and team members have voluntarily increased their lock-ups. Locked tokens include tokens that otherwise would have been released and entered the circulating supply. This number also includes OGN that is locked up by users in Origin Deals.")}}
```
Comment (franckc) [inline templates/dashboard.html:101]: Should line 116 copy be updated to also mention Origin Deals users? Right now it only says "Investors & team members with locked tokens"

File: templates/dashboard.html [hunk 2]
```diff
@@ -180,6 +180,15 @@ <h3>{{gettext("Wallets excluded from circulating supply")}}</h3>
               </a>
             </div>
 
+            <div class="reserved-wallet">
+              <div class="wallet-name">{{gettext("Lukewarm Wallet")}}</div>
```
Comment (franckc) [inline templates/dashboard.html:184]: Is "Lukewarm wallet" how we want to officially and publicly call this wallet?

Comment (joshfraser) [reply templates/dashboard.html:184]: No, let's stick with calling it "Distribution Staging" as it's labeled on CoinGecko.

File: templates/dashboard.html (hunk 1 above)
Comment (joshfraser) [reply templates/dashboard.html:101]: +1 to including staked Origin Deals tokens in the count. We'll need to add an endpoint to share that data w/ the dashboard.

File: templates/dashboard.html (hunk 2 above)
Comment (shahthepro) [reply templates/dashboard.html:184]: Actually, we already have that wallet listed. My bad, I didn't check the address before adding. 😅

File: templates/dashboard.html (hunk 1 above)
Comment (shahthepro) [reply templates/dashboard.html:101]: Done.

**Pull request merged**